### DISCUSSION TITLE
Automatic Snap to Grid

### DISCUSSION THREAD

**Discussion by garv-shah** [rxhanson/Rectangle#825] [Ideas]
Hi! Thank you very much for maintaining this app, I love everything about it!

I was thinking of a possible feature addition, which would automatically snap newly open apps to the grid.
For example, if I opened Safari, instead of floating, it would automatically maximise, and if I now opened Messages, it would split both in half and put Safari on the right. etc, etc, allowing for automatic tiling.

Once again, thank you very much for the amazing app, I can't wait to see what comes in the future!

**Comment by rxhanson** [comment]
Thanks for the kind words!

I do have plans to add some auto-tiling behavior to Rectangle Pro (paid), although it's a bit further down on my roadmap. If anyone wants to give it a go for Rectangle, a good PR is definitely welcome.

**Comment by garv-shah** [comment]
Alright, thanks, I'll see what I can come up with tonight!
